packages/terra-compact-interactive-list/src/subcomponents/Cell.jsx

```diff
@@ -71,14 +76,19 @@ const Cell = (props) => {
     columnMaximumWidth,
     widthUnit,
     onCellSelect,
+    isSelected,
     setFocusedCell,
     isRowHeader,
   } = props;
 
   const theme = useContext(ThemeContext);
   const cellRef = React.useRef();
   const [isSelectableCell, setIsSelectableCell] = useState(false);
-  const className = cx('cell', theme.className, { selectable: isSelectableCell });
+  const selectable = column.isSelectable && isSelectableCell;
+  const className = cx('cell', theme.className, {
+    selected: selectable && isSelected,
```

Comment:
I don't think the selected state should depend on the cell being selectable.  I could programmatically select a cell without allow a user to do so.

Reply:
In Compact Interactive list the cell should become non-selectable if after the component rendered it was detected that cell has intractable elements in it (like buttons). As that becomes known only after the cell was rendered, I am not sure how to change that, an example would be appreciated.

Reply:
If the consumer sets the isSelected prop, won't the cell be selected regardless?

Reply:
No, if the cell has an interactable element (like button) inside, it will not be selectable per design from @eawww